documentation/zones.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Zone support in vSphere Cloud Provider
+---
+
+## Introduction
+
+In the vSphere infrastructure, kubernetes nodes are run as VMs on an ESX host and Volumes are created as VMDK files on a datastore. The Volume is always placed on a datastore that is shared among all the node VMs of the k8s cluster. This is done so that the Volume can be attached to a pod scheduled on any of the node VMs. However, this implies that there should be a shared datastore among the node VMs in vSphere infrastructure to create Volumes. It also implies that the datastore capacity of any other non-shared datastores cannot be utilized for creating Volumes.
```

Comment:
Can you replace "kubernetes" => "Kubernetes"  all places. also "k8s" => "Kubernetes"

Reply:
Done.